src/data/styles/common.scss

```diff
@@ -0,0 +1,23 @@
+.pad-l-12 {
```

Comment:
When I read the class name I thought it would add padding not margin, this could cause confusion.